CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.11.0
+* Updated Rokt SDKs and interfaces, support partner-triggered close and show image on dark mode.
```

Comment:
partner-triggered close not supported yet in flutter sdk :)

Reply:
oops